src/Sarif.Viewer.VisualStudio.UnitTests/TestUtilities.cs

```diff
@@ -0,0 +1,24 @@
+﻿
+using Microsoft.CodeAnalysis.Sarif;
+using Microsoft.Sarif.Viewer.ErrorList;
+
+namespace Microsoft.Sarif.Viewer.VisualStudio.UnitTests
+{
+    public static class TestUtilities
+    {
+        public static void InitializeTestEnvironment()
+        {
+            // Creating a SarifViewerPackage object has the side effect of setting a static
```

Comment:
>Creating a SarifViewerPackage [](start = 15, length = 29)

would it be less expensive to just expose those static items from the package and force init? or are complicated things constructed that would make this difficult?